code/modules/vehicles/mecha/equipment/tools/work_tools.dm

```diff
@@ -310,7 +310,7 @@
 					return
 				F.PlaceOnTop(/turf/closed/wall)
 			else if(isopenturf(target))
-				var/turf/open/space/S = target
+				var/turf/open/S = target
```

Comment:
Since this no longer refers to "space", can you change this to `open_turf`?

Reply:
Yup